docs/user/customization.md

```diff
@@ -12,14 +12,53 @@ The most simple customization is exposed by the installer as an interactive seri
 
 While the default cluster size may be sufficient for some, many will need to make alterations. This can include increasing the number of machines in the control plane, changing the type of the virtual machines that will be used (e.g. AWS instances), or adjusting the CIDR range used for the Kubernetes service network. This level of customization is exposed via the installer's `install-config.yaml`. The install-config can be accessed by running `openshift-install create install-config`. This file can then be modified as needed before running a later target.
 
-The `install-config.yaml` generated by the installer will not have all of the available fields populated, so they will need to be manually added if they are needed. The full list of available fields can be found in the [Go Docs][godocs]. Documentation for each of the supported platforms can be found in their platform-specific section:
+The `install-config.yaml` generated by the installer will not have all of the available fields populated, so they may need to be manually added if they are needed.
+
+### Platform-agnostic properties
+
+The following `install-config.yaml` properties are supported on all platforms:
+
+* `apiVersion` (required string): The API version for the `install-config.yaml` content.
+    The current version (as described in this documentation) is `v1`.
+    The installer may also support older API versions.
+* `additionalTrustBundle` (optional string): a PEM-encoded X.509 certificate bundle that will be added to the nodes' trusted certificate store.
+* `baseDomain` (required string): The base domain to which the cluster should belong.
+* `controlPlane` (optional [machine-pool](#machine-pools)): The configuration for the machines that comprise the control plane.
+* `compute` (optional array of [machine-pools](#machine-pools)): The configuration for the machines that comprise the control plane.
```

Comment:
copy/paste error, shouldn't say `control plane`. Perhaps `The configuration for the worker nodes`? (or compute nodes)?

Reply:
> copy/paste error, shouldn't say `control plane`.

Fixed in 2c86ce813 -> 96b16e9d4.